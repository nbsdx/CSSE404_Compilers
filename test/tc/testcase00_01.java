// class redeclaration

class A
{
   public static void main( String[] args )
   {
      int b = (new A()).start();
   }
}

class A
{
   public int start()
   {
      return 3;
   }
}

