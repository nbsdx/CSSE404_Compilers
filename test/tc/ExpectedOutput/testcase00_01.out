Class named A already exists.
